pkg/deploy/deploy.go

```diff
@@ -175,7 +180,31 @@ func (d *deployer) Deploy(ctx context.Context) error {
 		}
 	}
 
-	return nil
+	return d.saveRPVersion()
+}
+
+// saveRPVersion for current location in shared storage account for environment
+func (d *deployer) saveRPVersion() error {
+	keys, err := d.accounts.ListKeys(context.Background(), *d.config.Configuration.GlobalResourceGroupName, *d.config.Configuration.RPVersionStorageAccountName, "")
```

Comment:
Using ListAccountSAS is preferred for security because it returns a time limited token.  You'll need something similar to what's in `func (m *manager) getBlobService`

Reply:
Done

Reply:
(See https://github.com/Azure/ARO-RP/pull/1033/commits/ea3fb19db90a80974ffcfec33d0fb673adb8f32d to review just this change)